distribution/lib/Standard/Database/0.2.28-SNAPSHOT/THIRD-PARTY/org.apache.httpcomponents.httpclient-4.5.13/NOTICES

```diff
@@ -0,0 +1 @@
+regarding copyright ownership.  The ASF licenses this file
```

Comment:
This is wrong. It is some random line out of context.

It was probably saved using "Keep" while "Keep with context" should have been used. In some situations "Keep in context" cannot be used, so you need to do "Ignore" and manually add the text including all relevant context to `copyright-add` file. As explained in the license review process docs: 
![image](https://user-images.githubusercontent.com/1436948/131677794-c4ea881b-6f4d-4f60-8178-2c6b3157c64c.png)